src/11-usart/buffer-overrun.md

```diff
@@ -3,7 +3,6 @@
 If you wrote your program like this:
 
 ``` rust
-#![deny(unsafe_code)]
```

Comment:
Can we keep these here and elsewhere?

Reply:
Yeah, I also hope so. But the method [`bits`](https://docs.rs/stm32f3xx-hal/0.6.1/stm32f3xx_hal/stm32/usart1/tdr/struct.TDR_W.html#method.bits) is not safe here. Could you give me some suggestions?

Reply:
Ah, sorry I missed that. There is no way around it then. These examples are too low-level. Thanks!

Reply:
My pleasure.